guides/plugins/plugins/storefront/fetching-data-with-javascript.md

```diff
@@ -0,0 +1,76 @@
+# Fetching data with JavaScript
+
+## Overview
+
+When you develop your own plugin, you might want to fetch data from the API which you need. This guide explains how to `
+achieve that.
+
+## Prerequisites
+
+While this is not mandatory, having read the guide about adding custom javascript plugins beforehand might help you understand this guide a bit further:
+
+{% page-ref page="./add-custom-javascript.md" %}
```

Comment:
These `page-ref` links might still be broken, like in my guides. 🙈  I would leave it in here, but we should keep in mind that we might to fix them later.